src/transports/tcp/TcpDuplexConnection.cpp

```diff
@@ -64,20 +76,20 @@ class TcpReaderWriter : public ::folly::AsyncTransportWrapper::WriteCallback,
     if (auto socket = std::move(socket_)) {
       socket->close();
     }
+    if (auto outputSubscription = std::move(outputSubscription_)) {
```

Comment:
Good catch. Because the closeErr method is a new method, which should also be updated. Updating now.